lib/linguist/languages.yml

```diff
@@ -3745,6 +3756,17 @@ QMake:
   - qmake
   ace_mode: text
   language_id: 306
+Quake:
+  type: programming
+  filenames:
+  - m3makefile
+  - m3overrides
+  extensions:
+  - ".tmpl"
```

Comment:
The original post is missing a search link for this extension.

Reply:
I’ll add any missing info later.